docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -755,8 +755,9 @@ Register table
 --------------
 The connector can register existing Iceberg tables with the catalog.
 
-An SQL procedure ``system.register_table`` allows the caller to register an existing Iceberg
-table in the metastore, using its existing metadata and data files::
+The SQL procedure ``system.register_table`` allows the caller to register an
```

Comment:
How about changing to just "The ``system.register_table`` procedure ...". I don't feel `SQL` is necessary here.